### ISSUE TITLE
Crash when launching game

### ISSUE THREAD

**Issue by Bneboy** [MrTJP/ProjectRed#1294]
Hey,
I get a crash when I launch the game after I updated a bunch of mods in my pack.

https://pastebin.com/EeNjskC2

Seems to be on ProjectRed's side, though I could be wrong. If I am wrong, please direct me where I should post this crash. Thanks.

**Comment by matix931** [comment]
Try to replace Project Red All modules pack with separated modules.

**Comment by covers1624** [comment]
Yeah, don't use the all jar, it's untested and exists for the build process, you should be using the seperate modules.
